detectron2/export/caffe2_export.py

```diff
@@ -64,10 +67,10 @@ def _check_eval(module):
             onnx_model = onnx.load_from_string(f.getvalue())
 
     # Apply ONNX's Optimization
-    all_passes = onnx.optimizer.get_available_passes()
+    all_passes = onnxoptimizer.get_available_passes()
     passes = ["fuse_bn_into_conv"]
```

Comment:
I'm pretty sure torch.onnx.export does this optimization so I think we can just remove onnxoptimizer entirely from this code:
https://cs.github.com/pytorch/pytorch/blob/9bb21fac9535f2eaec69f3c5bcf5617f7368b940/torch/csrc/jit/passes/onnx/eval_peephole.cpp#L39